src/hoodie/request.js

```diff
@@ -43,14 +42,17 @@ var exports = module.exports = function(hoodie) {
 //
 var API_PATH = '/_api';
 
-exports.request = function(hoodie, type, url, options) {
+exports.request = function(hoodie, method, url, options) {
+  //passing in an XHR instance just lets us test it
   var defaults = {
-    type: type,
-    dataType: 'json'
+    method: method,
+    json: true
```

Comment:
What happens, if I want to send a request that is not content-type: application/json? e.g. the `POST /_session` request from `account.signIn(username, password)` is such a request. Does it still work for you? It should send `Content-Type:application/x-www-form-urlencoded; charset=UTF-8`, could you please double check?